automated/linux/modules/modules.sh

```diff
@@ -81,6 +81,16 @@ scan_dmesg_for_errors() {
 	dmesg -l 0,1,2,3,4,5 | grep -E "BUG:|WARNING:|Oops:|Call Trace:" && report_fail "dmesg_error_scan" || report_pass "dmesg_error_scan"
 }
 
+check_module_unloaded() {
+	local _module="$1"
+	if lsmod | grep "^${_module}"; then
```

Comment:
Might want to add a awk/cut here to lessen the risk of false positives

Reply:
I'll update to this:
lsmod | grep "^${_module} " > /dev/null